README.md

```diff
@@ -105,4 +134,4 @@ As rudimentary ways to do vertical and horizontal arrangements, you can do `Shif
 To give us feedback, suggestions, to get help, or to submit a bug report, please visit the [Discord server](https://discord.gg/XxrzyphTPd).
 
 # Additional Credits
-Camera mirroring code derived from pyvirtualcam - https://github.com/letmaik/pyvirtualcam
\ No newline at end of file
+Camera mirroring code derived from pyvirtualcam - https://github.com/letmaik/pyvirtualcam
```

Comment:
Not sure why it's marking this line as changed other than perhaps the Github Readme editor forces it to have a newline at the end.